jfix-stdlib-ratelimiter/src/main/kotlin/ru/fix/stdlib/ratelimiter/RateLimiterKt.kt

```diff
@@ -0,0 +1,31 @@
+package ru.fix.stdlib.ratelimiter
+
+import java.time.temporal.TemporalUnit
+
+interface RateLimiterKt : AutoCloseable {
+
+    /**
+     * Get configured rate limit
+     */
+    suspend fun getRate(): Double
+
+    /**
+     * Try to acquire permit without waiting for it to become available.
+     *
+     * @return `true` if permit has been acquired `false` otherwise
+     */
+    suspend fun tryAcquire(): Boolean
+
+    /**
+     * Try to acquire permit. If it isn't immediately available waits for it specified timeout.
+     *
+     * @return `true` if permit has been acquired `false` otherwise
+     */
+    suspend fun tryAcquire(timeout: Long, unit: TemporalUnit?): Boolean
+
+    /**
+     * Update configuration
+     */
+    suspend fun updateRate(permitsPerSecond: Int)
```

Comment:
Is there any reason to make method for configuring rate limit suspendable?

Reply:
Removed suspend